voluptuous/tests/tests.py

```diff
@@ -1437,9 +1437,25 @@ def test_any_with_discriminant():
         schema({
             'implementation': {
             'type': 'C',
-            'c-value': None,}  
+            'c-value': None,}
         })
     except MultipleInvalid as e:
         assert_equal(str(e),'expected bool for dictionary value @ data[\'implementation\'][\'c-value\']')
     else:
         assert False, "Did not raise correct Invalid"
+
+def test_maybe_returns_subvalidator_error():
+    schema = Schema(Maybe(Range(1, 2)))
+
+    # The following should be valid
+    schema(None)
+    schema(1)
+    schema(2)
+
+    try:
+        # Should trigger a UrlInvalid exception
```

Comment:
Comment incorrectly says `UrlInvalid`, but I think should say `MultipleInvalid`.

Reply:
Hi!

Thank you for taking the time to review this!

I have addressed the comment, and rebased the commit on upstream/master.